src/client/dispatch.rs

```diff
@@ -337,13 +337,6 @@ async fn handle_event(
             // This could potentially be annoying to handle when otherwise wanting to normally take care of a new channel.
```

Comment:
This comment should be removed as it doesn't apply anymore.

Reply:
Done.